thunder/dynamo/report.py

```diff
@@ -796,6 +796,8 @@ def __str__(self):
     def run_benchmark(self, compile_fn: CompileSpecificationInterface, timer_fn: TimerInterface):
         compiled_fn = compile_fn.compile(self.nvfusion_bsym)
         inputs = self.make_example_inputs()
+        if compile_fn.name == "nvfuser":
```

Comment:
This is OK for now; in the future we probably want to let the compile specifications define a function that gets call on the inputs so that people can add their own specifications for nvfuser